pkg/sql/information_schema.go

```diff
@@ -262,6 +263,29 @@ https://www.postgresql.org/docs/9.5/infoschema-applicable-roles.html`,
 	},
 }
 
+var informationSchemaCharacterSets = virtualSchemaTable{
+	comment: `character sets available in the current database
+` + docs.URL("information-schema.html#character_sets") + `
+https://www.postgresql.org/docs/9.5/infoschema-character-sets.html`,
+	schema: vtable.InformationSchemaCharacterSets,
+	populate: func(ctx context.Context, p *planner, dbContext *dbdesc.Immutable, addRow func(...tree.Datum) error) error {
+		return forEachTableDesc(ctx, p, dbContext, hideVirtual, func(
```

Comment:
there should only be one for each database in this instance, based on my comparison with postgres

Reply:
Correct, fixed.